### DISCUSSION TITLE
cp showing 404 error

### DISCUSSION THREAD

**Discussion by Yu1ch** [statamic/cms#11061] [Q&A]
Hi, I recently started to test out statamic to build a website. I am using Google Cloud VMs to run an Ubuntu server, and managed to get the landing page working, but when I access /cp a 404 error shows up. I am quite stuck so any solutions would really help:)

![image](https://github.com/user-attachments/assets/bdd51022-7785-45ec-ae68-f6c0fb353e51)

**Comment by Yu1ch** [comment]
I managed to find a fix...
I had to set AllowOverride in my apache config file...

**Reply by Amarjit** [reply]
Looks like you are using Apache. Also ensure to also enable the "rewrite" module.
